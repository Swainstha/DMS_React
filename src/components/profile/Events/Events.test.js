import testSnapFunction from '../../../util/testSnapFunction';
import Events from './index'
import React from 'react';


testSnapFunction("<Events />", "Snapshot test for Events ",<Events /> )