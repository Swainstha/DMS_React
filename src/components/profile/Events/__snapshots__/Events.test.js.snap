// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Events /> Snapshot test for Events  1`] = `ShallowWrapper {}`;
